core/utils/cuckoo_map.h

```diff
@@ -501,12 +518,14 @@ class CuckooMap {
   }
 
   // Resize the space of buckets, and rehash existing entries
+  template <typename VV>
   void ExpandBuckets(const H& hasher, const E& eq) {
     CuckooMap<K, V, H, E> bigger(buckets_.size() * 2, entries_.size());
 
-    for (const auto& e : *this) {
-      // While very unlikely, this insert() may cause recursive expansion
-      bool ret = bigger.Insert(e.first, e.second, hasher, eq);
+    for (auto& e : *this) {
+      // While very unlikely, this Insert() may cause recursive expansion
```

Comment:
trivial: this isn't `Insert()` any more, it's `DoEmplace` now.

Overall this looks fine, but the whole "VV might be `V&&`, or might be `const V&`" thing (where `V` is already also a template) gives me a headache.